server/api/auth.filters.js

```diff
@@ -1,21 +1,28 @@
 const mustBeLoggedIn = (req, res, next) => {
   if (!req.user) {
-    return res.status(401).send('You must be logged in')
+    return res.status(401).send('You must be logged in');
   }
-  next()
-}
+  next();
+};
 
 const selfOnly = action => (req, res, next) => {
   if (req.params.id !== req.user.id) {
-    return res.status(403).send(`You can only ${action} yourself.`)
+    return res.status(403).send(`You can only ${action} yourself.`);
   }
-  next()
-}
+  next();
+};
 
 const forbidden = message => (req, res, next) => {
-  res.status(403).send(message)
-}
+  res.status(403).send(message);
+};
+
+const adminOnly = (req, res, next) => {
+  if (!req.params.isAdmin) {
```

Comment:
Assuming that the above 'req.user' works, I think this would work as 'req.user.isAdmin'